src/components/Toast/toast.st.css

```diff
@@ -0,0 +1,117 @@
+.toast-region {
+  position: fixed;
```

Comment:
The position of the region feels like something that can be controlled via prop? You could argue that it is something you pick once for a theme... Is is possible you might want  more than one ToastRegion on a page?

Reply:
Bottom by default I think as well.

Reply:
For some reason I remembered they suggested top-end. Seems like I did not remember well.

<img width="515" alt="Screenshot 2023-05-26 at 11 59 34 AM" src="https://github.com/action-is-hope/shelley/assets/16628652/c4c74022-e99c-43de-b534-d900faf26770">